_includes/landing-page/landing-description.html

```diff
@@ -0,0 +1,68 @@
+<div class="container">
+    <div class="columns is-multiline">
+        <div class="column is-12-tablet is-6-desktop is-spaced-bottom">
+            <h1 class="title is-size-3-tablet is-size-2-desktop has-text-grey-darker">
+                Run functions anywhere enabling youwith the same unified experience.
+            </h1>
+            <h1 class="title has-text-weight-light is-size-5 is-size-4-desktop has-text-grey">
+                Bring your laptop, your own on-prem hardware or create a cluster in the cloud.
+                Pick Kubernetes or Docker to do the heavy lifting enabling you to build a scalable,
+                fault-tolerant event-driven serverless platform for your applications.
+            </h1>
+            <h1 class="title has-text-weight-light is-size-5 is-size-4-desktop has-text-grey">
+                You can try out OpenFaaS in 60 seconds or write and deploy your first Python function in around 10-15 minutes.
+                From there you can take the OpenFaaS workshop, a series of tried-and-tested
+                self-paced labs which teach you everything you need to know about functions — and more.
+            </h1>
+        </div>
+        <div class="column is-12-tablet is-5-desktop is-offset-1-desktop">
+            <div class="columns is-multiline">
+                <div class="column is-12 is-spaced-bottom">
+                    <div class="columns">
+                        <div class="column is-3-tablet is-narrow-desktop has-text-centered-mobile">
+                            <p><img src="{{ site.baseurl }}/images/openfaas/store.png"></p>
+                        </div>
+                        <div class="column">
+                            <h1 class="title is-size-4 has-text-grey-darker">
+                                Functions Store
+                            </h1>
+                            <h1 class="title has-text-weight-light is-size-5-tablet is-size-4 has-text-grey">
+                                An ecosystem for sharing, reusing andcollaborating on functions
+                            </h1>
+                        </div>
+                    </div>
+                </div>
+                <div class="column is-12 is-spaced-bottom">
+                    <div class="columns">
+                        <div class="column is-3-tablet is-narrow-desktop has-text-centered-mobile">
+                            <p><img src="{{ site.baseurl }}/images/openfaas/template.png"></p>
+                        </div>
+                        <div class="column">
+                            <h1 class="title is-size-4 has-text-grey-darker">
+                                Templating system
+                            </h1>
+                            <h1 class="title has-text-weight-light is-size-5-tablet is-size-4 has-text-grey">
+                                Reduce boilerplate code, share code inthe templates store
```

Comment:
another missed space (you must be on a macbook pro :wink: )

![Screenshot from 2019-04-17 19-53-27](https://user-images.githubusercontent.com/12687192/56330239-dc1fef80-614c-11e9-9935-f42e8dca0867.png)